### DISCUSSION TITLE
Can LabGym detect behaviors like head twitch?

### DISCUSSION THREAD

**Discussion by abeWustl** [umyelab/LabGym#163] [Q&A]
I've used LabGym before to measure behaviors during forced swim and I'm wondering if LabGym is able to detect head twitch response and other "uncoordinated" locomotor behavior? My goal is to compared the behavioral differences between multiple pharmacological treatments which means that I need to train a model that will have to detect multiple behaviors like grooming, paw licking, horizontal locomotion, head twitch response, ataxia, and a few more. I attached a clip that shows the head twitch response as well as uncoordinated locomotion. Thanks!
 
[vlc-record-2024-07-02-10h54m45s-HeadTwitch_Ket.mp4-.zip](https://github.com/user-attachments/files/16071204/vlc-record-2024-07-02-10h54m45s-HeadTwitch_Ket.mp4-.zip)

**Comment by yujiahu415** [comment]
Yes, I think LabGym can definitely detect the head twitch and uncoordinated locomotion. But Let me know if you have any difficulties or questions when you use LabGym for this task. I'm happy to help you for any troubleshooting. Thanks!
